orion/controllers/announce/announceModel.go

```diff
@@ -0,0 +1,15 @@
+package announce
+
+import (
+	"database/sql"
+)
+
+type Announce struct {
+	Id				uint
+	CreatedAt		uint
+	UpdatedAt		uint
+	DeletedAt		sql.NullInt64
+	AnnounceId		string
+	Title			string
+	Message			string
+}
```

Comment:
This might be my mistake because I did not clearly communicate this:
The Announce struct should follow what the database columns look like.
In mysql, if you ` use mathnavdb` and then `describe announcements`, you'll see its properties:
```
+------------+------------------+------+-----+---------+----------------+
| Field      | Type             | Null | Key | Default | Extra          |
+------------+------------------+------+-----+---------+----------------+
| id         | int(10) unsigned | NO   | PRI | NULL    | auto_increment |
| created_at | bigint(20)       | NO   |     | NULL    |                |
| updated_at | bigint(20)       | NO   |     | NULL    |                |
| deleted_at | datetime         | YES  |     | NULL    |                |
| posted_at  | bigint(20)       | NO   |     | NULL    |                |
| author     | varchar(255)     | NO   |     | NULL    |                |
| message    | text             | NO   |     | NULL    |                |
+------------+------------------+------+-----+---------+----------------+
```

Here, the `Id` is just the row number and we do not need an `announceId`. Unlike `programs` where all the endpoints API use the String `programId`, for announcements, we will just use `id` (unsigned integer).
In addition, we won't need a `Title` (yet) and we'll need a `postedAt` and `author` column.
Sorry for the miscommunication.